python/cudf/cudf/io/parquet.py

```diff
@@ -622,6 +622,7 @@ def read_parquet(
         storage_options=storage_options,
         bytes_per_thread=bytes_per_thread,
         prefetch_options=prefetch_options,
+        kvikio_remote_io=cudf.get_option("kvikio_remote_io"),
```

Comment:
My understanding is that this feature is independent of the file format (e.g. parquet). Can we avoid adding this argument to `ioutils.get_reader_filepath_or_buffer` for now and just rely on the `"kvikio_remote_io"` option/configuration within that function?

I suppose we *may* want a key-word argument eventually, but I don't see much reason to add it now (even your read_parquet test just sets the `"kvikio_remote_io"` option in a context).

Reply:
Good point, removed